../../cc/trees/proxy_main.cc:156:20: error: constexpr variable 'num_buckets' must be initialized by a constant expression
  156 |   constexpr size_t num_buckets = 1 << begin_main_frame_reason.size();
      |                    ^             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


--- a/cc/trees/proxy_main.cc
+++ b/cc/trees/proxy_main.cc
@@ -153,7 +153,7 @@ void ProxyMain::RecordBeginMainFrameMetr
     const BeginMainFrameReasons& begin_main_frame_reason,
     const base::ElapsedTimer& timer,
     std::string_view suffix) const {
-  constexpr size_t num_buckets = 1 << begin_main_frame_reason.size();
+  const size_t num_buckets = 1 << begin_main_frame_reason.size();
 
   base::UmaHistogramCustomMicrosecondsTimes(
       base::StrCat({"Compositing.BeginMainFrame.TimeUs", suffix}),
